pkg/autoscaler/fake/fake_metric_client.go

```diff
@@ -85,6 +86,11 @@ func (t *MetricClient) StableAndPanicRPS(key types.NamespacedName, now time.Time
 	return t.StableRPS, t.PanicRPS, err
 }
 
+// CandidatesForRemoval returns names of pods which are candidate for removal
+func (t *MetricClient) CandidatesForRemoval(key types.NamespacedName, readyCount, desiredScale int) ([]string, error) {
+	return t.RemovalCandidates, nil
```

Comment:
Do we need these arguments `key, readyCount, desiredScale`?
As we are not using inside function

Reply:
This is a fake implementing an interface. The real implementation of the function makes use of those values.

Reply:
if you don't need them, remove the names `f(int,float,string)`.